third-party/realdds/src/dds-device-impl.cpp

```diff
@@ -345,9 +353,12 @@ bool dds_device::impl::init()
                     TYPE2STREAM( fisheye, video )
                     TYPE2STREAM( confidence, video )
                     TYPE2STREAM( pose, motion )
-                        DDS_THROW( runtime_error, "stream '" + stream_name + "' is of unknown type '" + stream_type + "'" );
+                    DDS_THROW( runtime_error, "stream '" + stream_name + "' is of unknown type '" + stream_type + "'" );
 
-                    if( default_profile_index < 0 || default_profile_index >= profiles.size() )
+                    if( default_profile_index >= 0 && default_profile_index < profiles.size() )
+                        stream->init_profiles( profiles, default_profile_index );
+                    else if( std::dynamic_pointer_cast< dds_video_stream >( stream ) ||
```

Comment:
The more I see, the more I dislike a topic per stream...

Reply:
Implemented as a single topic of the device (not of type `dds_stream/_server`)